projects/collection-editor-library/src/lib/services/config/label.config.json

```diff
@@ -250,5 +250,8 @@
       "info" : {
          "001" : "QR code image generation is in progress. Please try downloading after sometime"
       }
-    }
+    },
+    "categoryLabels": {
+      "multiselectMultipleChoiceQuestion": "Multiple Choice Question"
```

Comment:
Make it `"Multiselect Multiple Choice Question": "Multiple Choice Question"`. Keeping in ming that we should maximum avoid hardcoding anything in the code level.